hyppo/conditional_independence/tests/test_FCIT.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
```

Comment:
what's the difference between the previous test file and this one?

Reply:
see above